mantidproject/mantid
# Raw data bugs in muon analysis

**Description of work.**
In muon analysis the rebin option was overriding the original data and the `Plot raw` checkbox was being ignored.



**To test:**
Open Muon Analysis and load some data
Change the binning to fixed and pick a large number (e.g.20)
Toggling the `Plot raw` button will cause the plot to update, the binned data should only show up when it is toggled off

Go to the fitting tab
Add a function
Make sure `Plot raw` is ticked
Make sure `fit to raw` is ticked

Do a fit
The fit line and data lines should be to the raw data

Untick the `Plot raw`
Do a fit
The fit line should have more data points than the raw data

Untick the `Fit to raw`
Do a fit
The fit and data lines should have less binning

Tick the `Plot raw` 
Do a fit
The fit should have less data points than the data

Also check it carries over to sequential fitting correctly. 




Fixes #28906. 



In the release notes
---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
